src/index.ts

```diff
@@ -52,3 +53,5 @@ export type {
     RenderRowActionsProps,
     TableActionsSettings,
 } from './types/RowActions';
+
+export type {ColumnDef, Column, Cell, Header, TableSize} from './components/Table/types';
```

Comment:
How do these types relate to the types in `src/tanstack.ts`?
Can these types be used in the base table? If so, it seems like they could be made shared.

Reply:
These types extend types from tanstack. I guess no, they shouldn't be used for base table, but are needed for styled table on the usage side. Do you suggest importing them directly like from `build/esm...`?